# Description of configuration files can be found at "https://gitlab.ethz.ch/ics-group/projects/andrea/crs-2.0/-/wikis/ros-configuration"
type: "lowpass"
pub_rate: 50.0 # Hz
measurement_timeout_threshold: 2.0 # seconds

# initial state
initial_state:
  type: "pacejka_car"
  value: [0, 0, 0, 0.5, 0, 0]

sensors:
  sensor_names: ["vicon"]
  vicon:
    key: vicon # optional, if not set us e name of sensor

b_dx: [0.0029, 0.0087, 0.0087, 0.0029]
a_dx: [1.0000, -2.3741, 1.9294, -0.5321]

b_dy: [0.0029, 0.0087, 0.0087, 0.0029]
a_dy: [1.0000, -2.3741, 1.9294, -0.5321]

b_dyaw: [0.0029, 0.0087, 0.0087, 0.0029]
a_dyaw: [1.0000, -2.3741, 1.9294, -0.5321]

b_yaw: [0.0029, 0.0087, 0.0087, 0.0029]
a_yaw: [1.0000, -2.3741, 1.9294, -0.5321]